lib/fog/openstack/core.rb

```diff
@@ -172,6 +180,11 @@ def setup(options)
           instance_variable_set "@#{openstack_param}".to_sym, value
         end
 
+        @idempotent_verbs ||= %w(GET DELETE)
+        @idempotent ||= options[:idempotent].nil? ? false : !!options[:idempotent]
+        @retry_interval ||= options[:retry_interval] ? options[:retry_interval] : 5 # Seconds
+        @retry_limit ||= options[:retry_limit] ? options[:retry_limit] : 2
+
```

Comment:
Just a nit, but if I'm not mistaken, 3 lines above could be written in shorter way (also matching to pattern few lines bellow)
```ruby
@idempotent     ||= !!options[:idempotent]
@retry_interval ||= options[:retry_interval] || 5 # Seconds
@retry_limit    ||= options[:retry_limit] || 2
```

Reply:
That's neat.

Absolutely, I'lll looking into adding some tests

Reply:
@aufi, sorry didn't get the bandwidth to do it.